### PR TITLE
Providing notebooks and code for synthetic data generationUpdate tutorials new data

### PR DESCRIPTION
**Pull request by komashk**
Description of changes:

Two AWS blogs describing PyDeequ capabilities have been updated to use synthetic data instead of the original Amazon Reviews dataset:

[Testing data quality at scale with PyDeequ](https://aws.amazon.com/blogs/big-data/testing-data-quality-at-scale-with-pydeequ/) (updates are public), also [tutorial](https://github.com/awslabs/python-deequ/blob/master/tutorials/test_data_quality_at_scale.ipynb).
[Monitor data quality in your data lake using PyDeequ and AWS Glue](https://aws.amazon.com/blogs/big-data/monitor-data-quality-in-your-data-lake-using-pydeequ-and-aws-glue/) (blog reviews have been completed, awaiting publication).
Created a new folder under ./tutorials/synthetic_data to host Jupyter notebooks that describe data generation for the blogs mentioned above (datasets for product in Electronics and Jewelry categories) and for 18 other product categories.

The synthetic data is hosted publicly in s3://aws-bigdata-blog/generated_synthetic_reviews/data/.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tutorials/synthetic_data/01-synthetic-data-electronics.ipynb [hunk 1]
```diff
@@ -0,0 +1,2191 @@
+{
```
Comment (chenliu0831) [inline tutorials/synthetic_data/01-synthetic-data-electronics.ipynb:1]: <p>Is it necessary to mention "classic"?  I think DataScience 3.0 kernel is quite universal across all SageMaker notebook offering. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (chenliu0831) [inline tutorials/synthetic_data/01-synthetic-data-electronics.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    from essential_generators import DocumentGenerator
</pre><p>You could have a cell to install those if you want to pin the version</p>

<pre class="ql-syntax" spellcheck="false">!python3 -m pip install awswrangler==3.7.2
!python3 -m pip install ssential_generators==1.0
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (komashk) [reply tutorials/synthetic_data/01-synthetic-data-electronics.ipynb:1]: Neither SageMaker JupyterLab (newest SageMaker IDE) or the original SageMaker Notebooks provide DataScience 3.0 kernel. DataScience 3.0 kernel is specific to SageMaker Classic.

Given the basic Python libraries we used for data generation, I don't believe data generation experience or results will be different in different SageMaker offerings, but we wanted to document what we used. 

I can drop that line if it doesn't seem necessary.

Comment (komashk) [reply tutorials/synthetic_data/01-synthetic-data-electronics.ipynb:1]: Will add a cell to demonstrate version pinning.

Comment (chenliu0831) [reply tutorials/synthetic_data/01-synthetic-data-electronics.ipynb:1]: Sounds good. I think they might be using a universal container now.

**Review by chenliu0831 — APPROVED** [review]

**Pull request merged**